owca/containers.py

```diff
@@ -191,35 +349,46 @@ def sync_containers_state(self, tasks) -> Dict[Task, Container]:
             # Calculate inverse relastion of task_id to res_group name based on mon_groups_relations
             for ctrl_group, container_names in mon_groups_relation.items():
                 for container_name in container_names:
-                    container_name_to_mon_group[container_name] = ctrl_group
-            log.debug('container_name_to_mon_group: %s',
-                      pprint.pformat(container_name_to_mon_group))
+                    container_name_to_resgroup_name[container_name] = ctrl_group
+            log.debug('container_name_to_resgroup_name: %s',
+                      pprint.pformat(container_name_to_resgroup_name))
 
         # Create new containers and store them.
         for new_task in new_tasks:
-            container = Container(
-                new_task.cgroup_path,
-                rdt_enabled=self.rdt_enabled,
-                rdt_mb_control_enabled=self.rdt_mb_control_enabled,
-                platform_cpus=self.platform_cpus,
-                allocation_configuration=self.allocation_configuration,
-            )
+            # @TODO temporarily to make owca work
+            enable_multicontainer = True
+            if (enable_multicontainer and hasattr(new_task, "sub_cgroup_paths") and
```

Comment:
task should have sub_cgroup_paths

Reply:
will be fixed by adding field to Task class